README_zh_CN.md

```diff
@@ -151,23 +207,21 @@ alias find='mc find'
 ```
 
 ### Shell自动补全
-你也可以下载[`autocomplete/bash_autocomplete`](https://raw.githubusercontent.com/minio/mc/master/autocomplete/bash_autocomplete)到`/etc/bash_completion.d/`，然后将其重命名为`mc`。别忘了在这个文件运行source命令让其在你的当前shell上可用。
-
-```sh
-sudo wget https://raw.githubusercontent.com/minio/mc/master/autocomplete/bash_autocomplete -O /etc/bash_completion.d/mc
-source /etc/bash_completion.d/mc
-```
+如果你正在使用bash或者zsh，当前的shell将会自动地设置为使用补全特性来为你的mc命令进行建议或自动补全。
 
 ```sh
 mc <TAB>
-admin    config   diff     ls       mirror   policy   session  update   watch
-cat      cp       events   mb       pipe     rm       share    version
+admin    config   diff     find     ls       mirror   policy   session  sql      update   watch
+cat      cp       event    head     mb       pipe     rm       share    stat     version
 ```
 
 ## 了解更多
 - [MinIO Client完全指南](https://docs.min.io/docs/minio-client-complete-guide)
```

Comment:
These links all go to EN versions, do we have CN versions available?

Reply:
Yes, we do. I'll replace these links  to EN versions with those which go to CN versions. Thank you very much.